main.cpp

```diff
@@ -19,13 +19,22 @@ using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter):rwrtr(rewriter) {
         // Your code goes here
     };
 
     void run(const MatchFinder::MatchResult &Result) override {

```

Comment:
This code does not work as expected for the following example:
```cpp
// test.cpp
int main() {
    float fooooo;
    int i = (int)   fooooo   ;
    return 0;
}

Reply:
All problems are fixed.